gui/itemStats.py

```diff
@@ -856,7 +856,7 @@ def OnRightClick(self, event):
         elif 'wxMac' in wx.PlatformInfo:
             os.system("open " + file_)
         else:
-            subprocess.call(["xdg-open", file_])
+            subprocess.call({"xdg-open": file_})
```

Comment:
Yeah, I think pyCharm is wrong here.

Well, not exactly wrong.  It was complaining because  there were situations where we could pass in a `None`, and because `subprocess.call` supports `**kvargs` it throws it off a bit.  So I suspect that we could actually do it either way.

I went ahead and reverted it though.